src/styles/_defaults.scss

```diff
@@ -36,18 +36,23 @@ button {
 
 ul,
 ol {
-  margin-bottom: 30px;
-  padding-left: 0;
+  list-style-position: inside;
+  list-style-type: default;
+  list-style: default;
+  margin-bottom: 10px;
+  padding-left: 5;
```

Comment:
5px?

Reply:
Thanks. Thought that was a part of it, but it's not.

I was trying to solve the issue where the bullets on the list would not appear on the screen/were aligned further left than the rest of the text:
![image](https://user-images.githubusercontent.com/10962974/147629419-5cfd23cd-4fc3-471d-bb88-e7ca7897d0d4.png)

I added margin-left to the body to confirm this:
![image](https://user-images.githubusercontent.com/10962974/147629441-c32f0093-96f0-49d4-8b33-28e6db07e3f9.png)

But, yeah. all I needed was the 'list-style-position' to get the bullets inside the dang thing:
![image](https://user-images.githubusercontent.com/10962974/147629478-450b869a-66c9-4760-a6fd-fa30d81e849c.png)

Reply:
updated in 210ea7e

Reply:
Gotcha. You'll want to add some margin on the `li` elements, not the `ol` or `ul` since they're the containers. If you want to do multi-level embeds, you'll want to do something like:

```css
ul,
ol {
  margin-bottom: 30px;
  padding-left: 0;
}

ul ul,
ol ol {
  margin-left: 1.5em;
}

li {
  font-weight: 300px;
  font-size: 16px;
  max-width: 560px;
  margin-bottom: 10px;
  margin-left: 2em;
}
```

[Check this doc for why `em` would be better in this case](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Values_and_units#relative_length_units)

Reply:
Updated! 4ab3903

Thanks for the reminder about `em`. Don't work in css enough hahah

Was throwing everything at ul/ol because I thought the issue was with the formatting of the unordered list object itself. Didn't occur to me that it'd be something better left to the line item formatting. Guess I should read up on best practices for that.